x-pack/plugins/security_solution/public/common/components/endpoint/host_isolation/endpoint_host_isolation_status.tsx

```diff
@@ -28,12 +28,20 @@ export interface EndpointHostIsolationStatusProps {
 export const EndpointHostIsolationStatus = memo<EndpointHostIsolationStatusProps>(
   ({ isIsolated, pendingIsolate = 0, pendingUnIsolate = 0, 'data-test-subj': dataTestSubj }) => {
     const getTestId = useTestIdGenerator(dataTestSubj);
-    const isPendingStatuseDisabled = useIsExperimentalFeatureEnabled(
+    const isPendingStatusDisabled = useIsExperimentalFeatureEnabled(
       'disableIsolationUIPendingStatuses'
     );
 
+    const wasReleasing = useRef<boolean>(false);
+    const wasIsolating = useRef<boolean>(false);
+
+    useEffect(() => {
```

Comment:
I was not aware you could actually leave off the second argument to `useEffect()` when the dependencies are static 👍

Reply:
That's a mistake. Fixed 6058831878fe7e812043848dd8110e226aa8e85e. Only want to recompute on prop change and not on every render.